backends/kv/writer.go

```diff
@@ -181,7 +185,20 @@ func (a *Appender) Add(frame frames.Frame) error {
 			}
 		}
 
-		input := v3io.PutItemInput{Path: a.tablePath + fmt.Sprintf("%v", key), Attributes: row, Condition: condition}
+		key := indexVal(r)
+		// Add key column as an attribute
+		row[indexName] = key
+
+		var sortingVal interface{}
+		if len(indices) > 1 {
+			sortingFunc, err := a.funcFromCol(indices[1])
+			if err != nil {
+				return err
+			}
+			sortingVal = sortingFunc(r)
+			row[indices[1].Name()] = sortingVal
```

Comment:
can use `row[schema.SortingKey] = sortingVal`

Reply:
to use schema i need to do  "newSchema.(*v3ioutils.OldV3ioSchema).SortingKey", i prefer the current way

Reply:
ok.. you still can do this once in the beginning of the function and avoid redundant function calls on every iteration.. but 🤷‍♂

Reply:
it needs to be inside the loop as the value is calculated with the loop index (r)